.github/workflows/ci_tests.yaml

```
@@ -62,14 +62,19 @@ jobs:
             isDraft: true
           - os: windows-latest
             isDraft: true
-        # Pair Python 3.10 with NumPy 1.23 and Python 3.12 with NumPy 1.26
-        # Only install optional packages on Python 3.12/NumPy 1.26
+        # Pair Python 3.10 with the minimum supported versions of NumPy/Pandas/Xarray
+        # and Python 3.12 with the latest versions of NumPy/Pandas/Xarray
+        # Only install optional packages on Python 3.12
         include:
           - python-version: '3.10'
             numpy-version: '1.23'
+            pandas-version: '1.5'
+            xarray-version: '2022.03'
```

Try setting the equal-to sign here
```suggestion
            pandas-version: '=1.5'
            xarray-version: '=2022.03'
```